ourpackage/consultado.py

```diff
@@ -0,0 +1,87 @@
+from colorama import Fore
+import os
+
+def kirby():
+    while True:
+        os.system("cls")
+        print(Fore.GREEN + "    SOMETHING ABOUT ME")
+        print(Fore.RESET + "\n 1. Personal information")
+        print(" 2. My Favorite Quotes")
+        print(" 3. Motivational Story")
+        print(" 4. Comment from Barcelos")
+        print(" 5. Comment from Veslino")
+        print(" 6. Comment from Andaya")
+        print(Fore.RED + "\n                    0 - Exit")
```

Comment:
I would like to add a comment about your module - Piadozo

Reply:
You have done it.